defaults/ekscluster-kfd-v1alpha2.yaml

```diff
@@ -142,6 +142,7 @@ data:
         retentionSize: 120GB
         storageSize: 150Gi
       alertmanager:
+        defaultRules: "enabled"
```

Comment:
I would like this field to be `boolean` and not string, and the key should be installDefaultRules, like we did on kyverno/gatekeeper

Reply:
I got an error when on "reducer" when try to put a boolean